tests/index.test.ts

```diff
@@ -45,12 +57,38 @@ describe('sdk tests', () => {
       runtime.application = originalRuntime.application;
       runtime.origin = originalRuntime.origin;
 
+      window.clearTimeout = originalClearTimeout;
       window.setTimeout = originalTimeout;
       window.addEventListener = originalEventListener;
       oauthService.openPopup = originalOpenPopup;
       jest.restoreAllMocks();
     });
 
+    describe('Outreachcontext initialized', () => {
+      it('will resolve context promise when success', async () => {
```

Comment:
Can we change this to something that describes the test better? Like: "Supported email context parameters are populated"